analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -42,9 +41,13 @@ import org.ossreviewtoolkit.analyzer.PackageManagerResult
 import org.ossreviewtoolkit.analyzer.managers.utils.NpmDependencyHandler
 import org.ossreviewtoolkit.analyzer.managers.utils.NpmModuleInfo
 import org.ossreviewtoolkit.analyzer.managers.utils.expandNpmShortcutUrl
+import org.ossreviewtoolkit.analyzer.managers.utils.fixDownloadUrl
 import org.ossreviewtoolkit.analyzer.managers.utils.hasNpmLockFile
 import org.ossreviewtoolkit.analyzer.managers.utils.mapDefinitionFilesForNpm
-import org.ossreviewtoolkit.analyzer.parseAuthorString
+import org.ossreviewtoolkit.analyzer.managers.utils.parseAuthors
+import org.ossreviewtoolkit.analyzer.managers.utils.parseLicenses
+import org.ossreviewtoolkit.analyzer.managers.utils.parseVcsInfo
+import org.ossreviewtoolkit.analyzer.managers.utils.splitNamespaceAndName
```

Comment:
I don't like that these function names have no NPM reference, although the functions are NPM specific. Some, but not all of them, at least mention NPM in the documention. I think this could easily lead to accidental use in the wrong context, for example `fixDownloadUrl` sounds like a generic function.
I would prefer to rename them to e.g. `fixNpmDownloadUrl`, note that the other already imported from `NpmSupport` all have an NPM reference in their name.

Reply:
> the other already imported from `NpmSupport` all have an NPM reference in their name.

I guess by `NpmSupport` you mean `NodeSupport`? But this actually raises the question on my side whether we should rename `NodeSupport` to `NpmSupport`, as these functions are indeed NPM (= package manager) related, and not generically Node(.js) related. Also, other support classes / files also use the exact name of the package manager as the prefix.

Reply:
Yes, I agree that `NodeSupport` is the wrong name.